skimage/filters/tests/test_lpi_filter.py

```diff
@@ -1,12 +1,21 @@
 import numpy as np
+import scipy
 from numpy.testing import assert_, assert_equal
 from skimage._shared import testing
+from distutils.version import LooseVersion as Version
 import unittest
 
 from skimage import data
 from skimage.filters import LPIFilter2D, inverse, wiener
+from skimage._shared._warnings import expected_warnings
 
 
+if (Version(np.__version__) >= '1.15.0' and
+        Version(scipy.__version__) <= '1.1.0'):
```

Comment:
Please, fix the indentation here.

Reply:
How would you like it to be fixed. Pep8 doesn't specify the "correct" way
```python
# No extra indentation.
if (this_is_one_thing and
    that_is_another_thing):
    do_something()

# Add a comment, which will provide some distinction in editors
# supporting syntax highlighting.
if (this_is_one_thing and
    that_is_another_thing):
    # Since both conditions are true, we can frobnicate.
    do_something()

# Add some extra indentation on the conditional continuation line.
if (this_is_one_thing
        and that_is_another_thing):
    do_something()
```
I picked the bottom one.